parser.y

```diff
@@ -8616,6 +8625,10 @@ ShowTargetFilterable:
 	{
 		$$ = &ast.ShowStmt{Tp: ast.ShowDatabases}
 	}
+|	"CONFIG"
```

Comment:
Wait, given that this is listed under `ShowTargetFilterable`, do we expect the following to work?

```sql
SHOW CONFIG LIKE 'foo%';
```

Reply:
Yes, `show config like '%log%'` and `show config where name='log.level'` both are ok.